File: Managerial/DAL/Models/Location.cs
using System.Collections.Generic;

namespace DAL.Models
{
    public class Location : AuditableEntity
    {
        public string LocationRow { get; set; }
        public string locationColumn { get; set; }
        public string LocationShelf { get; set; }
        public int? WarehouseID { get; set; }
        public int? WarehouseItemID { get; set; }

        public virtual ICollection<Warehouse> Warehouses { get; set; }
        public virtual ICollection<WarehouseItem> WarehouseItems { get; set; }
    }
}

File: Managerial/Managerial/ClientApp/src/app/models/warehouse/Location.ts
import { BaseModel } from "../BaseModel.model";

export class Location extends BaseModel {

  constructor(id?: number,locationRow?: string, locationShelf?: string, locationColumn?: string){
    super()
    this.LocationColumn = locationColumn;
    this.LocationShelf = locationShelf;
    this.LocationRow = locationRow;
    this.id = id;
  }
  public LocationRow: string;
  public LocationColumn: string;
  public LocationShelf: string ;
}
